dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -405,13 +413,37 @@ object LagomPlugin extends AutoPlugin {
   }
 
   private def runAllMicroservicesTask: Initialize[Task[Unit]] = Def.taskDyn {
+
     val projects = microservicesProjects.value
-    val filter = ScopeFilter(inProjects(projects: _*))
+    val filter: ScopeFilter = ScopeFilter(inProjects(projects: _*))
+    // check if application.conf is present in service
+    val serviceResourceDirectories = ((unmanagedResourceDirectories in Runtime) ?? Nil).all(filter)
+    //Verifying the presence of the required application.conf file
+    val verifyApplicationConfTask = Def.task {
+
+      val servicesPaths: Seq[File] = serviceResourceDirectories.value.flatten
+
+      val (validProject, notValidProject) = servicesPaths.partition(f => {
+        val r: PathFinder = {
+          f.get * "application.conf"
+        }
+        r.get.nonEmpty
+      })
+      //displays all projects that not contain a application.conf file
+      if (notValidProject.nonEmpty) {
+        notValidProject foreach (f => {
+          val log = state.value.log
+          log.error("File application.conf has been missed in the following service >>> " + f.project)
```

Comment:
Again you are right for the message,
My apologises, I am a new learner on this tools. I have some basic knowledges, but I work hard to tend to become an expert (thanks for your help) :D

it's better than what i done. Regarding the point which you speak after from my point of view, the developer must understand what he did. If he don't understand the purpose of the application.conf file, I propose to add a check on the fact that the key :"play.modules.enabled" exists and at least one value has affected to it.
With that, we can display a more complete message which explain the purpose of the file, also we can provide a link to the documentation that talk about this file.

One other way to solve this issue seems to be more complicated (with my knowledge on SBT's possibilities). I imagine that the next step is to scan the JVM's memory to detect the module that make the binding for this service (?) but I don't know. 

What do you think? 

If you agree, I can prepare au new PR with the proposal above (the first one).